en/flight_modes/mission.md

```diff
@@ -27,22 +27,26 @@ Missions are uploaded onto a SD card that needs to be inserted **before** bootin
 
 At high level all vehicle types behave in the same way when MISSION mode is engaged:
 
-1. If a mission is stored and PX4 is flying it will execute the [mission/flight plan](../flying/missions.md) from the current step. 
-1. If a mission is stored and PX4 is landed:
-   * On copters PX4 will execute the [mission/flight plan](../flying/missions.md). If the mission does not have a `TAKEOFF` command then PX4 will fly the vehicle to the minimum altitude before executing the remainder of the flight plan from the current step.
-   * On fixed-wing vehicles PX4 will not automatically take off (the autopilot will detect the lack of movement and set the throttle to zero). The vehicle may start executing the mission if hand- or catapult- launched while in mission mode.  
-1. If no mission is stored, or if PX4 has finished executing all mission commands:
+1. If no mission is stored, or if PX4 has finished executing all mission commands, or if the [mission is not feasible](#mission-feasibility-checks):
    * If flying the vehicle will loiter.
    * If landed the vehicle will "wait".
+1. If a mission is stored and PX4 is flying it will execute the [mission/flight plan](../flying/missions.md) from the current step.

```

Comment:
@sfuhrer @Igor-Misic What if the current step is 0 and the first item is takeoff and you're already flying? On MC the vehicle will just skip takeoff and start at the next item. On FW I assume the same thing?

Otherwise I assume it just starts from the first item after takeoff if you are already flying?

Reply:
I don't think it will skip the Takeoff WP for MC, but consider it as a normal WP. And it currently does the same for FW. 
I will propose to change it on the PX4 side, as I would also expect it to not go to the Takeoff if already in air.

Reply:
It actually behaves differently for FW and MC flight: in MC it treats the Takeoff WP as a normal WP, in FW it only has to be above the Takeoff altitude to consider it reached. No special logic is in place for already taken off or not, so it will do the same whether it already is in air when the mission is started or not.

Reply:
Hmmm. Don't know what that actually means for FW. I assume it means it will fall out of the air for a vehicle configured to catapult launch since it won't have acceleration required to ramp up the throttle. I guess we could test it. 

I'll just add the not for MC